### DISCUSSION TITLE
How to find iPXE server on network?

### DISCUSSION THREAD

**Discussion by SysAdminNoob** [ipxe/ipxe#1150] [Q&A]
Hello all,

I'm in the process of implementing Windows imaging through WDS. After configuring WDS, I wanted to execute a test run through VM, instead of PXE boot loading, I'm presented with iPXE (1.21.1). I spoke with my IT Ops manager about this and he was not aware of this system being installed nor utilized. The person who had installed this system did not leave any documentation / SOPs. Neither one of us are familiar with iPXE, unfortunately.

According to our research, the IP addresses it's using are not active/assigned, the only thing it does have correct is the DNS server. It does seem to provide a mac address as well - tried running arp -a command to see if its listed, but that's a no go. We discovered this information by hitting CTRL+B, then going into config. 

And my assumption is that our iPXE is sitting on a VM.

In the configuration settings screen, it lists, "innotek GmbH" as the manufacturer and, "VirtualBox" as the product. However, this is all in a VirtualBox environment, so I don't know if iPXE is just displaying that it's running in the VirtualBox session that I've created or if it's literally sitting in a VirtualBox VM hosted somewhere in our network. 

Anyone have any idea how I would go about finding what machine is hosting iPXE? As of now, this is a high alert for our team. It could compromise our security - we haven't had any compromises _yet_.

**Comment by NiKiZe** [comment]
So you are using VirtualBox, they are using iPXE for their VM boot.
If so, there is 2 stages of Ctrl-B, the ROM, and the boot.
What happens if you just let the machine boot?

Or do you get iPXE on every machine you are trying to boot on the network?
If so you should use tcpdump or wireshark to figure out your traffic, especially the TFTP packets, and then ask your switch which port that MAC address is on.

**Reply by SysAdminNoob** [reply]
So I goofed.. 

I assume this has to do with the fact that DHCP and WDS are on two whole different servers, thus iPXE not being able to find the WDS server and the boot image. 

But after further research I learned that iPXE comes built in with VirtualBox. iPXE is not installed somewhere on any of our servers. I had tested a physical machine and it's able to load PXE (not iPXE) just fine, just need to make configuration changes between WDS/DHCP/Router. 

My apologies for wasting anyone's time.
